### DISCUSSION TITLE
Is bull and bullmq related?

### DISCUSSION THREAD

**Discussion by aishadeshmukh** [OptimalBits/bull#2355] [Q&A]
Is bull a legacy version of bullmq? Does bullmq provides latest features? Or are these 2 separate things entirely?

**Comment by Manuel2u** [comment]
@aishadeshmukh 
please did you get an answer?
or do you know the answer?

**Reply by roggervalf** [reply]
hey sorry for the delay. Yes bull is a legacy versión. Currently in maintenance mode. We are not planing to add more features to bull. Instead we are recommending to migrate to bullmq

**Reply by Manuel2u** [reply]
Oh okay
Thank you
